### ISSUE TITLE
Cosyvoice3 example.py例子生成的句子完全乱读

### ISSUE THREAD

**Issue by ruaruaruabick** [FunAudioLLM/CosyVoice#1692]
example.py中cosyvoice3_example()第一个例子（其他例子也是同样问题）：
[zero_shot_0.wav](https://github.com/user-attachments/files/24206271/zero_shot_0.wav)
```
def cosyvoice3_example():
    cosyvoice = AutoModel(model_dir='pretrained_models/Fun-CosyVoice3-0.5B-2512')
    # zero_shot usage
    for i, j in enumerate(cosyvoice.inference_zero_shot('八百标兵奔北坡，北坡炮兵并排跑，炮兵怕把标兵碰，标兵怕碰炮兵炮。', 'You are a helpful assistant.<|endofprompt|>希望你以后能够做的比我还好呦。',
                                                        './asset/zero_shot_prompt.wav', stream=False)):
        torchaudio.save('zero_shot_{}.wav'.format(i), j['tts_speech'], cosyvoice.sample_rate)


def main():
    cosyvoice3_example()


if __name__ == '__main__':
    main()
```

**log如下**
failed to import ttsfrd, use wetext instead
2025-12-17 14:16:27,599 INFO input frame rate=25
/XXX/lib/python3.11/site-packages/torch/nn/utils/weight_norm.py:143: FutureWarning: `torch.nn.utils.weight_norm` is deprecated in favor of `torch.nn.utils.parametrizations.weight_norm`.
  WeightNorm.apply(module, name, dim)
2025-12-17 14:31:31.396526331 [W:onnxruntime:, transformer_memcpy.cc:111 ApplyImpl] 14 Memcpy nodes are added to the graph main_graph for CUDAExecutionProvider. It might have negative impact on performance (including unable to run CUDA graph). Set session_options.log_severity_level=1 to see the detail logs before this message.
2025-12-17 14:31:31.401025977 [W:onnxruntime:, session_state.cc:1316 VerifyEachNodeIsAssignedToAnEp] Some nodes were not assigned to the preferred execution providers which may or may not have an negative impact on performance. e.g. ORT explicitly assigns shape related ops to CPU to improve perf.
2025-12-17 14:31:31.401041204 [W:onnxruntime:, session_state.cc:1318 VerifyEachNodeIsAssignedToAnEp] Rerunning with verbose output on a non-minimal build will show node assignments.
0%|                                                                                                                                     | 0/1 [00:00<?, ?it/s]
2025-12-17 14:32:39,509 INFO synthesis text 八百标兵奔北坡，北坡炮兵并排跑，炮兵怕把标兵碰，标兵怕碰炮兵炮。
/XXX/CosyVoice/cosyvoice/cli/model.py:101: FutureWarning: `torch.cuda.amp.autocast(args...)` is deprecated. Please use `torch.amp.autocast('cuda', args...)` instead.

  with self.llm_context, torch.cuda.amp.autocast(self.fp16 is True and hasattr(self.llm, 'vllm') is False):
/data/megastore/Projects/liyuhan/codes/CosyVoice/cosyvoice/cli/model.py:406: FutureWarning: `torch.cuda.amp.autocast(args...)` is deprecated. Please use `torch.amp.autocast('cuda', args...)` instead.
  with torch.cuda.amp.autocast(self.fp16):
2025-12-17 14:32:48,673 INFO yield speech len 7.68, rtf 1.193300262093544
100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 1/1 [00:28<00:00, 28.29s/it]

**Comment by lymanzhao** [comment]
相同问题，极速复刻模式。

在自然语言控制没有问题，但语音效果实在不敢恭维……

**Comment by ruaruaruabick** [comment]
@lymanzhao 找到解决方案了，实测有效 #1677，不过很神奇为什么会和transformers版本强关联...
